plugins/README.md

```diff
@@ -36,6 +37,22 @@ Tips:
 -   To see a trace of logs and wasm ABI calls, add `--loglevel=TRACE`.
 -   Optionally specify plugin config data using the `--config=<path>` flag.
 
+# Building Plugins
```

Comment:
nit: have a blank line before and after each `#` section header, both for visual spacing when viewing the source, and consistency with existing content.